mindsdb_native/libs/phases/data_cleaner/data_cleaner.py

```diff
@@ -58,6 +58,13 @@ def run(self):
         self.transaction.lmd['zero_information_columns'] = zero_information_columns
         self.transaction.lmd['columns_to_ignore'].extend(empty_columns)
 
+        # Exclude target columns
+        self.transaction.lmd['columns_to_ignore'] = list(set(
+            self.transaction.lmd['columns_to_ignore']
+        ).difference(
+            self.transaction.lmd.get('predict_columns', [])
+        ))
```

Comment:
Would it not be better to instead *not* run the `unique identifier` analysis for the target columns instead ? Since in Max's bug, I assume that's where the issue is happening.

Reply:
it's not ran for `force_columns_usage[]` columns and for `predict_columns[]`
I decided to remove the `zero_information_columns` key because it kind of repeats the unique identifier detection logic